cpp/tests/transform/integration/unary_transform_test.cpp

```diff
@@ -599,9 +599,9 @@ TEST_F(StringOperationTest, Output)
 TEST_F(StringOperationTest, StringConcat)
 {
   auto first_name = cudf::test::strings_column_wrapper{
-    "John", "Mia", "Abd", "Mendes", "Arya", "John", u8"François", u8"José", u8"Søren", u8"张"};
+    "John", "Mia", "Abd", "Mendes", "Arya", "John", "François", "José", "Søren", "张"};
```

Comment:
@davidwendt I'm not confident this didn't change the inputs, do you know?
Verified that the last element here has size 3 as a string, and prints as shown here.

Had to remove us because in C++20 it means an array of char8_t, which can't be used to directly  create a `std::string`. u8 literals need to be made into `std::u8string` and than that is converted into `std::string` in a loop/transform. These steps seem as suspicions as just dropping the prefix.